### DISCUSSION TITLE
How do I disable DHT so it doesn't turn on when restarted?

### DISCUSSION THREAD

**Discussion by Issykul** [crazy-max/docker-rtorrent-rutorrent#269] [Q&A]
I've put `RT_DHT_PORT=0` into my `rtorrent-rutorrent.env` so far. But this doesn't disable DHT in the Config in the web-ui.

Is there a way for me to disable it so the checkbox isn't checked when the `docker-compose.yml` is re-deployed?

**Comment by Issykul** [comment]
Sorry for pinging you directly, but maybe you can help me @crazy-max

**Reply by kerryandjane** [reply]
in the .rc file there is this:
```
 #Enable DHT support for trackerless torrents or when all trackers are down
#May be set to "disable" (completely disable DHT), "off" (do not start DHT),
 #"auto" (start and stop DHT as needed), or "on" (start DHT immediately)
dht.mode.set = auto
```
give that a go.

Edit the settings on .rtorrent.rc and save it to /data/rtorrent/.rtorrent.rc and restart. See if the settings stick.
